src/profile-logic/function-info.js

```diff
@@ -11,8 +11,12 @@
  * it will return the original string.
  */
 export function stripFunctionArguments(functionCall: string): string {
-  // Remove known data that can appear at the end of the string
-  const s = functionCall.replace(/ \[clone [^]+\]$/, '').replace(/ const$/, '');
+  // Remove known data that can appear at the start or the end of the string
+  const s = functionCall
+    .replace(/ \[clone [^]+\]$/, '')
```

Comment:
Optional: Documenting each of these regexes with an example match would be a nice addition.

Reply:
I did it, thanks for the feedback!